index.html

```diff
@@ -19156,25 +19168,25 @@ <h2 id="AudioWorklet-Sequence">
             <a>AudioWorkletProcessor</a>.
           </p>
           <ol>
-            <li>In the main scope, <code>window.audioWorklet</code> is
-            requested to import a script. No <a>AudioWorkletGlobalScope</a>s
-            exist yet, so the script is fetched and added to the Worklet module
-            responses map.
-            </li>
             <li>An <a>AudioContext</a> is created.
             </li>
-            <li>An <a>AudioWorkletGlobalScope</a> is created in association
-            with the context's audio rendering thread. This is the global scope
-            in which <a>AudioWorkletProcessor</a> class definitions will be
-            evaluated.
+            <li>In the main scope, <code>context.audioWorklet</code> is
+            requested to add a script module.
+            </li>
+            <li>Since none exists yet, a new <a>AudioWorkletGlobalScope</a> is
+            created in association with the context's audio rendering thread.
+            This is the global scope in which <a>AudioWorkletProcessor</a>
+            class definitions will be evaluated.
             </li>
-            <li>As part of the global scope's initialization, the set of
-            imported scripts is run, including the one that was previously
-            imported.
+            <li>The imported script is run in the newly created global scope.
             </li>
             <li>As part of running the imported script, an
             <a>AudioWorkletProcessor</a> is registered under the key
             <code>"Custom1"</code> within the <a>AudioWorkletGlobalScope</a>.
+            This populates maps both in the global scope and in the
+            <a>AudioContext</a>.
```

Comment:
s/AudioContext/BaseAudioContext/

Here we have some cross-thread operations going on. Not sure if this captures the correct "order" of sequence with the proper spec language. ("queue a task" and all?)

Reply:
This is the non-normative section. I added some normative "queue a task" language in the newly added step 11 of the normative description of `registerProcessor()`; see here:

https://s3.amazonaws.com/pr-preview/WebAudio/web-audio-api/1435-context-owns-worklet.html#dom-audioworkletglobalscope-registerprocessor

Reply:
BTW I believe the usage of `AudioContext` is correct since this is a non-normative example and states it's not a general description but a specific narrative of one use case.

Reply:
Still LGTM! Thanks for this change! 👍